ADOPTERS.md

```diff
@@ -7,5 +7,8 @@ This really helps the project to gain momentum and credibility. It's a small con
 |--------------|---------|--------------------|
 | [Alliander](https://www.alliander.com/en/open-source/) | ospo@alliander.com | Alliander is mainly interested in the security side of the software supply chain. Alliander uses multiple tools and has evaluated even more, but currently ORT is unbeaten. ORT provides accurate and destailed scan results and allows us to define a through policy to use in evaluation. We are currently working to adopt ORT for thorough scans in addition to other tools. We're working with the community to get ORT running in GitHub Actions. |
 | [Deutsche Telekom AG](https://www.telekom.de/) | daniel.eder@magenta.at | Deutsche Telekom is in the process of adopting ORT as part of our CI/CD chain and is working on a custom ORT reporter in our [ORT fork](https://github.com/telekom/ort). |
-| [HERE Technologies](https://here.com/) | opensource@here.com | HERE Technologies uses ORT to generate SBOMs for all its products, as well as to manage licensing risks. |
+| [HERE Technologies](https://here.com/) | opensource@here.com | HERE Technologies uses ORT to generate SBOMs for all its products, as well as to manage licensing risks.
+|
+| [Porsche AG](https://www.porsche.com/) | Open_Source_Office@porsche.de | Porsche AG started using ORT in the year 2020 where we utilize ORT as a core service for FOSS Compliance and FOSS Vulnerability.Our approach is to run ORT as a service decentrally by decoupling logically Analyzer and Scanner where the Product Development Team takes the responsibility for running the analyzer and the open source program office takes the responsibility for running the scanner, curation, and report consolidation for auditing.
+|
```

Comment:
This pipe symbol needs to be put at the end of line 12 (separated with a single space from the `.` at the end of the preceding sentence, like it was originally in line 10).